### PR TITLE
Rename LightsOn to LightsOff

### PR DESCRIPTION
**Pull request by Thundermaker300**
Makes more sense, since the lights are generally on in all the rooms unless they are flickered off for some reason.

Plus the documentation ("flickered on") makes no sense

### CODE REVIEW

File: Exiled.API/Features/Room.cs [hunk 1]
```diff
@@ -146,15 +146,15 @@ public Color Color
         public IEnumerable<Camera> Cameras { get; private set; } = Enumerable.Empty<Camera>();
 
         /// <summary>
-        /// Gets or sets a value indicating whether or not the lights in this room are currently flickered on.
+        /// Gets or sets a value indicating whether or not the lights in this room are currently off.
         /// </summary>
-        public bool LightsOn
+        public bool LightsOff
```
Comment (iopietro) [inline Exiled.API/Features/Room.cs:151]:
```suggestion
        public bool AreLightsOff
```
For consistency with other bools.

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Room.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Room.cs:151]: Is not Are please :3

Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Room.cs]: > Is not Are please :3

???

Comment (Thundermaker300) [reply Exiled.API/Features/Room.cs]: Ignore him he's a bit goofy with English

**Comment by Thundermaker300** [comment]
Resigned

**Pull request merged**